3회차/권건희/4_신용카드만들기1.py

```diff
@@ -1,3 +1,18 @@
 import sys
 
 sys.stdin = open("_신용카드만들기1.txt")
+
+T=int(input())
+for i in range(T):
+    list_=list(map(int,input().split()))
+    odd=[]
+    even=[]
+    for l in range(1,len(list_)+1):
+        if l%2==0:
+            even.append(list_[l-1])
+        else:
+            odd.append(2*(list_[l-1]))           
+    a=(sum(odd)+sum(even))
```

Comment:
`even` 과 `odd` 두 리스트로 나누지 않는 풀이도 있어요!
```suggestion
    num = []
    for l in range(1,len(list_)+1):
        if l%2==0:
            num.append(list_[l-1])
        else:
            num.append(2*(list_[l-1]))           
    a=sum(num)
```